index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <main id="main">
+      <header id="title">
+        <h1>Dr. Norman Borlaug</h1>
+        <a>The man who saved a billion lives</a>
+      </header>
+
+      <div id="img-div">
+        <img id="image" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg">
+        <p id="img-caption">Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger.</p><br>
+      </div>
+
+      
+
+      <section>
```

Comment:
- [ ] Good job so far :+1:   I noticed that you add it blank lines. Kindly, remove those blank lines, [do not add blank lines, spaces, or indentations without a reason](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)